packages/camera/camera_android/android/src/main/java/io/flutter/plugins/camera/features/zoomlevel/ZoomLevelFeature.java

```diff
@@ -6,16 +6,18 @@
 
 import android.graphics.Rect;
 import android.hardware.camera2.CaptureRequest;
+import android.os.Build;
 import io.flutter.plugins.camera.CameraProperties;
 import io.flutter.plugins.camera.features.CameraFeature;
 
 /** Controls the zoom configuration on the {@link android.hardware.camera2} API. */
 public class ZoomLevelFeature extends CameraFeature<Float> {
-  private static final float MINIMUM_ZOOM_LEVEL = 1.0f;
+  private static final Float defaultZoomLevel = 1.0f;
```

Comment:
The previous naming style was correct per https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names.